projects/ad405x_iio/app/ad405x_iio.h

```diff
@@ -0,0 +1,35 @@
+/*************************************************************************//**
+ *   @file   ad405x_iio.h
+ *   @brief  Header for AD405X IIO application
+******************************************************************************
+* Copyright (c) 2023 Analog Devices, Inc.
```

Comment:
should this be 2024?

Reply:
Updated the copyright year for all files modified in 2024